cloudtools/start.py

```diff
@@ -70,7 +73,13 @@ def init_parser(parser):
     # initialization action flags
     parser.add_argument('--init', default='', help='Comma-separated list of init scripts to run.')
     parser.add_argument('--vep', action='store_true', help='Configure the cluster to run VEP.')
+    
```

Comment:
Added the ability to override the default project and region from the gcloud config